src/main/java/uk/gov/ons/ctp/common/event/model/ContactCompact.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.ons.ctp.common.event.model;
+
+import com.godaddy.logging.LoggingScope;
+import com.godaddy.logging.Scope;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class ContactCompact {
```

Comment:
Could we not make Contact extend ContactCompact in a similar way to Address and AddressCompact?
Avoids dupe fields and is consistent

Reply:
Changed